Automattic/woocommerce-payments
# Add tracking to user interaction of deposits settings

Fixes #5882 

#### Changes proposed in this Pull Request



Add tracking to user interactions to the deposits panel on the settings page.

![Screenshot 2023-05-11 at 3 28 26 PM](https://github.com/Automattic/woocommerce-payments/assets/57298/5c67b641-3769-4a0d-87df-ce1e479c7c76)


When a Merchant clicks on the "Manage in Stripe" link to update account details, the event  `wcpay_settings_deposits_manage_in_stripe_click` will be tracked



The link-based events use an `onClick` which may not be the optimal event. In testing, this works with both mouse, keyboard, and touch navigation.



#### Testing instructions

Visit the WCPay Settings Screen, scroll down to the deposits section.

* Click the Manage in Stripe link
* See event `wcpay_settings_deposits_manage_in_stripe_click` has been tracked



-------------------


**Post merge**
